src/main/java/com/beerjournal/breweriana/user/UserController.java

```diff
@@ -22,8 +21,11 @@
     }
 
     @GetMapping
-    ResponseEntity<Collection<UserDto>> getAllUsers() {
-        return new ResponseEntity<>(userService.getAllUsers(), HttpStatus.OK);
+    ResponseEntity<Page<UserDto>> getUsers(
+            @RequestParam(value = "page", defaultValue = "0") int page,
+            @RequestParam(value = "count", defaultValue = "10") int count,
+            @RequestParam(value = "firstName", defaultValue = "", required = false) String firstName) {
```

Comment:
With empty string it is working like without filtering, maybe this is a good idea to refactor collections items in similar way later?